src/components/CookieWarning.js

```diff
@@ -33,29 +61,43 @@ const CookieWarning = () => {
     }, cookies)
     signOut()
   }
-  return !cookiesAccepted && aside({
+
+  return h(Transition, {
+    in: showWarning,
+    timeout: { exit: animTime * 1000 },
+    mountOnEnter: true,
+    unmountOnExit: true
+  }, [transitionState => aside({
     'aria-label': 'cookie consent banner',
```

Comment:
Since the banner is no longer present when the page first loads, should this have an alert role or aria-live attribute to make sure that it's read by a screen reader?

Reply:
I'll give it an alert role - I think aria-live has to be applied to parent containing the element change, and technically, that would mean attaching it to the root div, which is probably too much to watch for changes on